controllers/index.js

```diff
@@ -94,4 +95,41 @@ module.exports = {
       message: "Email confirmed successfully",
     });
   },
+
+  sendLoginToken: async (req,  res, next) => {
+    const { email, password } = req.body;
+    let userData;
+    let loginToken;
+
+    try {
+      // finding user in db
+      const { user, userExist } = await checkUserExist(email);
+      if(!userExist) throw new ErrorResponse(401, "There is no account associated with this email, Signup now");
```

Comment:
### @shamilkotta When user information not found in database, probably it's a [**404 error**](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404).
```suggestion
      if(!userExist) throw new ErrorResponse(404, "There is no account associated with this email, Signup now");
```